index.bs

```diff
@@ -91,6 +91,10 @@ spec: credential-management-1; urlPrefix: https://w3c.github.io/webappsec-creden
     for: CredentialsContainer
         type: method
             text: create(); url: dom-credentialscontainer-create
+
+spec: mixed-content; urlPrefix: www.w3.org/TR/mixed-content/
```

Comment:
Could you add the `https://` to the front of this `urlPrefix`?

Reply:
Oh, I think you can remove this block entirely. You're using "a priori authenticated URL", which is exported from the spec.